src/ripe_rainbow/domain/base/assertions.py

```diff
@@ -19,7 +26,8 @@ def at_url(self, url):
         # iterates over the complete set of URLs to be tested and sees
         # if at least one of them validates as a prefix
         for _url in url:
-            if not current_url.startswith(_url): continue
+            no_params = urljoin(current_url, urlparse(current_url).path)

```

Comment:
what’s the rationale on this, URL checking? We cannot keep changing the semantics on the API ;)

Reply:
I'd argue the semantics are the same: check if the current URL matches the given one. Thing is before it was broken and would generate false positives (say it matches when in fact it shouldn't)